dietpi/dietpi-software

```diff
@@ -4185,6 +4193,16 @@ amvdec_vp9' > /etc/modules-load.d/dietpi-c4-kodi.conf
 
 			# Dependencies: https://github.com/pi-hole/pi-hole/blob/development/automated%20install/basic-install.sh#L250
 			G_AGI $PHP_NAME-xml $PHP_NAME-sqlite3 $PHP_NAME-intl
+      
+		# Unbound
+		if (( ${aSOFTWARE_INSTALL_STATE[182]} > 1 )); then
+			G_WHIP_MSG 'You are installing Pi-hole while Unbound is already installed.  The Unbound port will be changed to 5353.\n\nTo add it to Pi-hole, change one of your DNS servers to 127.0.0.1:5353.'
```

Comment:
Let's check if the file does not exist yet, before warning the user and creating it. E.g. Pi-hole might be reinstalled and hence unbound already running on port 5353 which makes the warning obsolete.